source/includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs

```diff
@@ -339,7 +339,6 @@ public static List<Guitar> SearchAfter()
 
         var result = guitarsCollection.Aggregate()
```

Comment:
Note to tech reviewer: this isn't the exact change the user suggested (see [ticket](https://jira.mongodb.org/browse/DOCSP-51915)), but it seems to work fine to return the full document. let me know if there's something I'm missing.